deep-learning/src/main/scala/ai/onnx/proto/Notice.md

```diff
@@ -0,0 +1,11 @@
+# OnnxMl.java
```

Comment:
Can you add some comment illustrating how to generate the code? Or even better, check in the script that generates the java code?

Reply:
The "script" is one line, and is just "protoc onnx-ml.proto" or something like that.  But yeah, I can add that explicitly.  To be honest, I did not even run it myself, I just grabbed the already-generated file from someone else's repo, but I know HOW to do it.